### PR TITLE
events: fix logspam from dashboard transaction spinner

### PR DESCRIPTION
**Pull request by benjamincburns**
addresses #5006

### CODE REVIEW

File: packages/events/defaultSubscribers/dashboard.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ module.exports = {
     "rpc:request": [
       function (event) {
         const { payload } = event;
-        if (payload.method === "eth_sendTransaction") {
-          // TODO: Do we care about ID collisions?
+        if (
+          isDashboardNetwork(this.config) &&
```
Comment (gnidan) [inline packages/events/defaultSubscribers/dashboard.js:13]: heh sure you can make this wrapper :)

Comment (gnidan) [reply packages/events/defaultSubscribers/dashboard.js:13]: by why not instead just check the negation at the very start and short circuit return early if it's not the dashboard network?

**Review by gnidan — COMMENTED** [review]
quick question before i approve

File: packages/events/defaultSubscribers/dashboard.js (hunk 1 above)
Comment (benjamincburns) [reply packages/events/defaultSubscribers/dashboard.js:13]: happy to do that as well - one sec.

**Review by gnidan — APPROVED** [review]
nice and clean!

**Pull request merged**